kernel/tests/s11_rackscale_benchmarks.rs

```diff
@@ -666,6 +670,368 @@ fn rackscale_vmops_benchmark(is_shmem: bool) {
     }
 }
 
+#[test]
+#[cfg(not(feature = "baremetal"))]
+fn s11_rackscale_shmem_vmops_unmaplat_benchmark() {
+    rackscale_vmops_unmaplat_benchmark(true);
+}
+
+#[cfg(not(feature = "baremetal"))]
+fn rackscale_vmops_unmaplat_benchmark(is_shmem: bool) {
+    use std::collections::HashSet;
+    use std::sync::Arc;
+    use std::thread::sleep;
+    use std::time::Duration;
+
+    let file_name = if is_shmem {
+        "rackscale_shmem_vmops_unmaplat_benchmark.csv"
+    } else {
+        "rackscale_ethernet_vmops_unmaplat_benchmark.csv"
+    };
+    let _ignore = std::fs::remove_file(file_name);
+
+    let build = Arc::new({
+        let mut build = BuildArgs::default()
+            .module("init")
+            .user_feature("bench-vmops-unmaplat")
```

Comment:
Is the only change from the the map test this user feature & the regex matching? Since there's so much duplication, maybe we could make an `is_unmap: bool` argument?

Reply:
Fixed!